src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -26,6 +33,7 @@ public class NodeAutoCompleteSearchViewModel : SearchViewModel
         private string noRecommendationsOrLowConfidenceTitle;
         private bool displayNoRecommendationsLowConfidence;
         private bool displayLowConfidence;
+        private double confidenceThreshold = 0.5;
```

Comment:
@varvaratou Which threshold are you referring to? This particular one is more for UI affordance, to hide results with less confidence etc, I would not expect end users to touch this often. I remember there is another threshold more related to service?

Reply:
I was referring to the threshold for the UI affordance - I was thinking more in terms of whether we have the flexibility to test it and see if this number makes sense and change if not. It may be a bit strict. I am thinking for example a case where the service could return 5 recommendations with scores 0.4, 0.2, 0.2, 0.1, 0.1. In this example, none would fall under the high confidence score but there would still be the top recommendations. 

Just an idea, maybe an alternative approach could be including the nodes whose confidence score sum up to a particular threshold.

Reply:
@varvaratou @reddyashish @QilongTang  it would be kind of interesting to consider pulling this value from launch darkly or a config, then you could roll out different confidence thresholds to % of users or change it whenever you want without a new release...

Reply:
Yes, we have an upcoming task to add this control in the preferences panel. Will add this information to that task.